.github/workflows/e2e-tests-ci.yml

```diff
@@ -1,10 +1,6 @@
-name: mattermost-e2e-tests
+name: E2E Tests
 on:
-  pull_request:
-  push:
-    branches:
-      - master
-      - mono-repo*
+  workflow_dispatch:
```

Comment:
If this is defined as workflow_dispatch, where is this called from? I can't seem to find that.

Reply:
Maybe this is a good opportunity to add comment or short description on how/where it will be triggered so it makes it easier for others to trace/follow along the hops of pipeline stages.